### ISSUE TITLE
The goal

### ISSUE THREAD

**Issue by Madis0** [Blaze4D-MC/baked-minecraft-models#1]
I know this is a work in progress, but could you explain in simpler terms what is the goal of this mod.

**Comment by hYdos** [comment]
faster entities?

**Comment by Madis0** [comment]
I figured something like that but _faster_ is still vague.

Improved FPS? Improved client-server data transfer? Improved movement smoothness (as you mentioned animations)?

**Comment by burgerindividual** [comment]
> I figured something like that but _faster_ is still vague.
> 
> Improved FPS? Improved client-server data transfer? Improved movement smoothness (as you mentioned animations)?

The initial goal of the project was to get better performance with Vulkan because uploading vertex data every frame was slowing us down considerably. Now, the goal is to better scale with rendering lots of entities on the screen at once on both platforms, even though Vulkan hasn't been implemented yet. The main place this mod improves on is uploading less data to the GPU.

**Comment by Madis0** [comment]
Right, so it improves FPS on client-side. Got the answer, thank you!
